### DISCUSSION TITLE
How to hide media control bar initially when video is not played. ?

### DISCUSSION THREAD

**Discussion by kevalrathod** [muxinc/media-chrome#985] [Q&A] [answered]
I am using the latest Media Chrome with Next.js app router. I want to hide the media control bar initially when the video has not been played yet. I tried using the following CSS:

```
media-control-bar:not([mediahasplayed]) {
   display: none;
}

/* Edit media control bar */
media-control-bar {
  background: var(--media-control-background, linear-gradient(#0000, #000000bf));
  padding-top: 20px;
  padding-bottom: 15px;
  padding-left: 15px;
  padding-right: 15px;
  gap: 8px;
  --media-font-size: 1.2rem;
}

/* Make the control bar buttons transparent using variables like --media-control-background */
media-control-bar > * {
  --media-control-background: transparent;
  --media-control-color: white;
  --media-button-padding: var(--media-button-padding);
  --media-control-padding: var(--media-controll-padding);
  --media-control-hover-background: var(--media-base-hover-background);
  border-radius: var(--media-hover-item-radius);
}

media-control-bar[mediahasplayed] {
   display: flex;
}
```

However, this hides the controller all the time, even after the video has played.

<img width="1429" alt="Screenshot 2024-09-26 at 12 22 28 AM" src="https://github.com/user-attachments/assets/ef529cb5-366a-43d6-a3d6-8ac6f1f2df46">

**Comment by luwes** [comment, marked as answer]
not all controls receive all the media attributes. try using

```css
media-controller:not([mediahasplayed]) media-control-bar {
  display: none;
}
```

inspiration at https://github.com/muxinc/elements/blob/main/packages/mux-player/src/themes/gerwig/gerwig.html

**Reply by kevalrathod** [reply]
Thank you very much it works. @luwes

Just one suggestion please try to explain more in documentation with  will help a lot.
